tests/make-genesis.nix

```diff
@@ -0,0 +1,6 @@
+{ cardano-sl-tools, runCommand }:
+
+runCommand "genesis" { buildInputs = [ cardano-sl-tools ]; } ''
```

Comment:
This could now call `./scripts/generate/genesis.sh` since it has nix support. It's important to the the genesis script anyway :)